### PR TITLE
Layout project footer

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# **Summary**
 
- I added the footer.

# **Trello Card**

https://trello.com/c/119vF7QB

# **Screeshots**

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,6 +105,12 @@ <h3 class="tech-release">Release: 2005</h3>
       </div>
     </div>
   </div>
+  
+  <div class="footer">
+    <h1 class="footer-title">WE ARE GAME CHANGERS</h1>
+    <img src="img/wolox_logo_white.svg" alt="footer-img" class="footer-img">
```
Comment (mcallegari10) [inline index.html]: I think `footer-img` is not a really good `alt`. Keep in mind search engines use the `alt` as a helper to crawl information about the page. Every image should have a meaningful `alt` value

File: index.html [hunk 2]
```diff
@@ -107,6 +105,12 @@ <h3 class="tech-release">Release: 2005</h3>
...
+    <h3 class="footer-copyright">Copyrigth 2018. All rights reserved</h3>
+  </div>
 
 </body>
-</html>
+</html>
```
Comment (mcallegari10) [inline index.html]: Remember to leave a blank EOF line

File: scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+.footer-title {
+  font-size: 24px;
+  color: white;
+  margin: 5px;
+}
+
+.footer-copyright {
+  color: white;
+  font-size: 12px;
+  margin: 10px 0 -10px 0 ;
```
Comment (mcallegari10) [inline scss/footer.scss]: Why are you using a negative value here?

File: scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.footer-title {
+  font-size: 24px;
+  color: white;
+  margin: 5px;
+}
+
+.footer-copyright {
+  color: white;
```
Comment (mcallegari10) [inline scss/footer.scss]: Use a SASS variable please 😄

File: scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.footer-img {
+  height: 60px;
+  margin: 5px;
+}
+
+.footer-title {
+  font-size: 24px;
+  color: white;
```
Comment (mcallegari10) [inline scss/footer.scss]: Use SASS variable for colors

File: scss/navbar.scss [hunk 6]
```diff
@@ -15,10 +15,10 @@
 }
 
 .nav-img {
-  height: 30px;
-  width: auto;
+    height: 30px;
+    width: auto;
```
Comment (mcallegari10) [inline scss/navbar.scss]: I think this spaces are not needed

File: scss/navbar.scss [hunk 7]
```diff
@@ -15,10 +15,10 @@
...
 }
 
 .nav-title {
-  font-size: 24px;
+    font-size: $text-big;
```
Comment (mcallegari10) [inline scss/navbar.scss]: Keep 2 spaces indentation

File: scss/tech.scss [hunk 8]
```diff
@@ -12,7 +12,7 @@
   border-radius: 5px;
   display: flex;
   flex-direction: column;
-  justify-content: space-around;
+  justify-content: space-evenly;
```
Comment (mcallegari10) [inline scss/tech.scss]: I think that's not a valid value. Why are you changing it?

File: scss/titles.scss [hunk 9]
```diff
@@ -18,3 +18,4 @@
     z-index: -1;
   }
 }
+
```
Comment (mcallegari10) [inline scss/titles.scss]: Leave a blank EOF line please

File: index.html (hunk 1 above)
Comment (mar7in91Wolox) [reply index.html]: Done!

File: index.html (hunk 2 above)
Comment (mar7in91Wolox) [reply index.html]: Done!

File: scss/footer.scss (hunk 3 above)
Comment (mar7in91Wolox) [reply scss/footer.scss]: I was trying to do anything that I don't remember now. Line cleaned :)

File: scss/footer.scss (hunk 4 above)
Comment (mar7in91Wolox) [reply scss/footer.scss]: Done!

File: scss/footer.scss (hunk 5 above)
Comment (mar7in91Wolox) [reply scss/footer.scss]: Done!

File: scss/titles.scss (hunk 9 above)
Comment (mar7in91Wolox) [reply scss/titles.scss]: Done!

**Comment by mar7in91Wolox** [comment]
All issues solved!

**Pull request merged**